provider/cldr/src/cldr_paths.rs

```diff
@@ -166,12 +174,16 @@ impl CldrPaths for CldrPathsAllInOne {
             .clone()
             .join(format!("cldr-misc-{}", self.locale_subset)))
     }
+    fn uprops(&self) -> Result<Option<PathBuf>, Error> {
+        Ok(self.uprops_root.clone())
+    }
 }
 
 #[cfg(test)]
 pub(crate) fn for_test() -> CldrPathsAllInOne {
```

Comment:
Suggestion (optional): Code cleanup: make this function pub and make it be `CldrPathsAllInOne::for_testdata()` and then call it from datagen.

Reply:
Actually, it looks like, based on CI, there are a lot of call sites.  So, I'm changing my suggestion: please change this back to the old name, but keep it pub.

Reply:
Actually, one more thing.  Since it looks like doing this requires that you move icu_testdata from dev-dependencies to dependencies, maybe just don't do this at all and revert to how it was before.

Reply:
Done